modules/ROOT/pages/index.adoc

```diff
@@ -3,12 +3,9 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Connectors facilitate integrations with third-party APIs. Modules are extensions to the Mule runtime that you can use when building a Mule application.
+Anypoint Connectors facilitate integrations with third-party APIs. Modules are extensions to the Mule runtime that you can use when building a Mule application. For a tutorial on how to use connectors, see the xref:4.2@::mule-runtime/mule-app-tutorial.adoc[Mule App Development Tutorial].
```

Comment:
The tutorial isn't for how to use connectors, it's for how to build an integration (and includes info about using connectors).  Maybe something like "The Mule app tutorial includes a section on adding the database connector to a sample Mule app" or something similar? Or you can just delete this sentence and add a link to the See Also section:
Mule App Tutorial (adding a database connector) or something. 

Can we add the relevant conceptual info from the old tutorial here?

Reply:
Changed to:
See xref:4.2@::mule-runtime/mule-app-tutorial.adoc[Mule App Development Tutorial] for information on adding a connector to a sample Mule app.